rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -5138,11 +5138,7 @@ fn install_does_not_reserve_cycles_on_system_subnet() {
     test.install_canister(
         canister_id,
         wat_canister()
-            .init(
-                wat_fn()
-                    .stable_grow((USAGE / WASM_PAGE_SIZE_IN_BYTES) as i32 - 1)
-                    .stable_read(0, 42),
```

Comment:
Benefit of the change: this test really wanted a stable_grow to happen, the read is irrelevant (but was forced due to the invalid Wasm).